lockbox-ios/Presenter/WelcomePresenter.swift

```diff
@@ -32,23 +33,36 @@ class WelcomePresenter {
     private weak var view: WelcomeViewProtocol?
 
     private let routeActionHandler: RouteActionHandler
+    private let settingActionHandler: SettingActionHandler
+    private let userInfoStore: UserInfoStore
     private let userDefaults: UserDefaults
+    private let biometryManager: BiometryManager
     private let disposeBag = DisposeBag()
 
     init(view: WelcomeViewProtocol,
          routeActionHandler: RouteActionHandler = RouteActionHandler.shared,
-         userDefaults: UserDefaults = UserDefaults.standard) {
+         settingActionHandler: SettingActionHandler = SettingActionHandler.shared,
+         userInfoStore: UserInfoStore = UserInfoStore.shared,
+         userDefaults: UserDefaults = UserDefaults.standard,
+         biometryManager: BiometryManager = BiometryManager()) {
         self.view = view
         self.routeActionHandler = routeActionHandler
+        self.settingActionHandler = settingActionHandler
+        self.userInfoStore = userInfoStore
         self.userDefaults = userDefaults
+        self.biometryManager = biometryManager
     }
 
     func onViewReady() {
-        let biometricButtonText = LAContext.usesFaceId ? Constant.string.signInFaceID : Constant.string.signInTouchID
-        let biometricImageName = LAContext.usesFaceId ? "face" : "fingerprint"
+        let biometricsAvailable = self.biometryManager.usesFaceID || self.biometryManager.usesTouchID
```

Comment:
Move this to the `biometryManager`.

`lazy var usesBiometrics: Bool = usesFaceID ||  usesTouchID`